### ISSUE TITLE
TBROK for ima_violation test

### ISSUE THREAD

**Issue by ZhangYet** [linux-test-project/ltp#923]
Found that the ima_violation test is broken due to the lack of /var/log/messages.

For example https://storage.kernelci.org/renesas/master/renesas-devel-2022-03-08-v5.17-rc7/arm64/defconfig+ima/gcc-10/lab-cip/ltp-ima-r8a774a1-hihope-rzg2m-ex.txt

Also, I occurred the same problem when I run the ltp ima test on a Ubuntu 20.04 server.

Does ima_violation need improvement?

**Comment by pevik** [comment]
Yes, there is an easy hack: https://github.com/linux-test-project/ltp/issues/372
Feel free to implement it.

**Comment by pevik** [comment]
https://patchwork.ozlabs.org/project/ltp/patch/20250114014319.547406-1-pvorel@suse.cz/
